RPI/Top/instances.fpp

```diff
@@ -313,18 +313,13 @@ module RPI {
   instance rateGroupDriverComp: Svc.RateGroupDriver base id 1700 \
   {
 
-    phase Fpp.ToCpp.Phases.configObjects """
+    phase Fpp.ToCpp.Phases.configComponents """
```

Comment:
I don't think we want to move the array declaration from the `configObjects` phase to the `configComponents` phase.

* The `configObjects` phase is for declaring static objects and has program-lifetime scope
* The `configComponents` phase is inside a function and has local function scope

Maybe it's OK to declare the array inside the function, if the array values get copied by the call, but this feels like a change we don't want to do.